stl/inc/atomic

```diff
@@ -131,6 +131,68 @@ _NODISCARD extern "C" bool __cdecl __std_atomic_has_cmpxchg16b() noexcept;
 #define ATOMIC_LLONG_LOCK_FREE    2
 #define ATOMIC_POINTER_LOCK_FREE  2
 
+#if _HAS_CXX20
+extern "C" {
+void __stdcall __std_atomic_wait_direct(const void* const _Storage, const void* const _Comparand,

```

Comment:
In declarations, we avoid marking value parameters as top-level const, because it's immediately ignored by the compiler and has no meaning to callers. (That is, this should be `const void* _Storage`.) Occurs repeatedly.